resources/etcd/INSTALL.md

```diff
@@ -1,67 +1 @@
 # Prerequisites
```

Comment:
Since we ended up removing everything from the Prerequisites, should we add here some text, so the tab is not empty?